### ISSUE TITLE
Prevent widget to add result to page

### ISSUE THREAD

**Issue by pgtaboada** [FriendlyCaptcha/friendly-challenge#216]
We are using js callbacks and our own js to check and handle captcha errors and results. Unfortunately, there is this whole automagic behaviour in the the background creating dom-elements, searching for elements to create widgets, etc.

Is there any way to turn all this automagic off and use the widget in full manual mode?

**Comment by gzuidhof** [comment]
Hi Papick,

If you import `friendly-challenge` as a library, there is no automatic searching for widgets to create on the page. You would import the library into your code as follows:

```javascript
import { WidgetInstance } from "friendly-challenge";

function doneCallback(solution) {
    console.log("CAPTCHA completed succesfully, solution:", solution);
    // ... Do something with the solution, maybe use it in a request
}

// This element should contain the `frc-captcha` class for correct styling
const element = document.querySelector("#my-widget");
const options = {
    doneCallback: doneCallback,
    sitekey: "<my sitekey>",
}
const widget = new WidgetInstance(element, options);

// this makes the widget fetch a puzzle and start solving it.
widget.start()
```

This example is from [this documentation page here](https://docs.friendlycaptcha.com/#/widget_api?id=if-you-are-using-the-friendly-challenge-library). There are a few more options you can pass in, which are documented below the code example: for example you can customize the name of the field, or disable any CSS injection (if you'd like to style the widget yourself entirely).

**Comment by gzuidhof** [comment]
If the question is more around preventing the the hidden form field from appearing at all - the widget doesn't support that currently.

I don't think we've ever had a request for that before (I imagine one could ignore the field value or put it outside of the `form` element instead), but of course if there is a real usecase for that then we can consider implementing that.

**Comment by pgtaboada** [comment]
We would appreciate if we could prevent the Widget of manipulating the DOM. We are trying to debug why the chrome password manager gets triggered when we have captcha on the login form. It is hard if the widget does things to the DOM we can't prevent.

**Comment by gzuidhof** [comment]
I am afraid that the widget will still manipulate the dom (to show the different states), it would stop changing the hidden form field.

I imagine that the Chrome password manager listens to changes of the form contents, but I don't have a guarantee that is how it behaves.

Anyway - I'm happy to implement a proof of concept in a branch, I imagine I can do it fairly quickly :)

**Comment by gzuidhof** [comment]
Implemented it here: https://github.com/FriendlyCaptcha/friendly-challenge/pull/217. 

You should be able to install this specific branch for testing like this:

```shell
npm install "https://github.com/FriendlyCaptcha/friendly-challenge.git#no-hidden-form-field"
```

**Comment by pgtaboada** [comment]
Great - can u give a hint no how to build it so we get the js to add it to our project. Not using npm.

**Comment by gzuidhof** [comment]
I have just released version 0.9.13 which includes this change. You can download the built JS files here: https://cdn.jsdelivr.net/npm/friendly-challenge@0.9.13/
